src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,6 +49,113 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants">
```

Comment:
If the Datasource is ProjectFile and the PersistedName is the same, then you can simply leave out the .DataSource section and have the XAML be simpler.